src/controllers/activegate/capability/capability.go

```diff
@@ -51,6 +60,7 @@ type Capability interface {
 	InitContainersTemplates() []corev1.Container
 	ContainerVolumeMounts() []corev1.VolumeMount
 	Volumes() []corev1.Volume
+	CreateService() bool
```

Comment:
Please rename it to `ShouldCreateService()` because it doesn't actually create the service.

Reply:
Done